docker-without-sudo.md

```diff
@@ -14,6 +14,14 @@ Optionally change the username to match your preferred user.
 $ sudo gpasswd -a $USER docker
 ```
 
+If is does not work, try this:
```

Comment:
This should describe in what cases the above command might not work and what the below command does.

Reply:
Also typo